backend/engine/plugins/repo_health/main.py

```diff
@@ -13,32 +15,32 @@ def main():
     output = {
         "success": False,
         "truncated": False,
-        "details": [],
+        "details": {},
```

Comment:
Yes, but it shouldn't be breaking. This PR is merging into the feature branch for repo health, not main.

This is how `inventory` plugins handle details, so I figured consistency there would be beneficial.

Reply:
But does it make sense to be a dictionary? Is it serving the same purpose as with `inventory` plugins?

Reply:
I'm not sure. What is the rational behind it being a dictionary in Inventory?

From what I can tell, the same thing applies to both inventory and configuration. Inventory has things broken up by the type of inventory (at least that's what it seems like by the technology discovery plugin). To me, it makes sense that Configuration would break things up by the type of configuration check (so, for example, if we had an AWS health check that would be in its own key).

I'm open to doing it however, though